build/commands/lib/config.js

```diff
@@ -1190,12 +1215,7 @@ Config.prototype.update = function (options) {
 Config.prototype.getTargetOS = function() {
```

Comment:
this function can now be removed in favor of direct use of `targetOS` variable, but it's better to do this in a separate PR.